Expensify/App
# Update profilePage to include timezone

### Details

PR does a couple of things: 

- Removes the now unnecessary `fetchTimezone()` function and updates the `personalDetails`onyx key so that each user's personalDetails also contains a timezone. 
- Updates the `priorityMode` onyx key to comply with how we want to save NVP onyx keys. 
- Updates the selectable area for opening the details page to be the avatar and the user's name (as requested [here](https://github.com/Expensify/Expensify.cash/pull/1593#issuecomment-796798141)
- Updates the Details page so that it now includes an abbreviated timezone name for clarity (ex. 10:51 AM PST instead of just 10:51 AM)

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes https://github.com/Expensify/Expensify/issues/156248

### Tests
For testing you're going to need two accounts: 
`Account A`: This an account who on a policy with multiple different users 
`Account B`: Someone who is on the same policy as account A. 

1. Log into E.com with Account A
2. Navigate to Settings > Account > Preferences > Timezone and modify your timezone 
3. Log into E.cash with Account A and check your `myPersonalDetails` onyx key to confirm it contains the correct timezone. 
4. In E.cash, send a message to Account B
5. Confirm the message has the correct timestamp for your timezone. 
6. Click on the name of Account B in the chat header (not on the avatar) and confirm it opens up the `Details` page. 
7. Confirm you see Account B's timezone in the details. 
8. In E.cash, navigate to Settings > Preferences and confirm the page loads as expected
9. Confirm you can modify your priority mode selection and that the `nvp_priorityMode` onyx key updates correctly. 

### Tested On

### Screenshots

#### Web
<img width="1200" alt="Screen Shot 2021-03-19 at 10 00 04 AM" src="https://user-images.githubusercontent.com/16747822/111819092-2f50ce80-889d-11eb-944f-93b0f109cc4a.png">

#### Mobile Web
<img width="200" alt="" src="https://user-images.githubusercontent.com/16747822/111819171-47285280-889d-11eb-9d12-d10818e009ac.png">

#### Desktop
<img width="900" alt="Screen Shot 2021-03-19 at 10 26 03 AM" src="https://user-images.githubusercontent.com/16747822/111819393-83f44980-889d-11eb-8143-6cc8c2944737.png">

#### iOS
<img width="200" alt="" src="https://user-images.githubusercontent.com/16747822/111819229-57403200-889d-11eb-8b4c-c3ae0c56765f.png">

#### Android
<img width="200" alt="" src="https://user-images.githubusercontent.com/16747822/111819270-61623080-889d-11eb-8ebf-31b788e056b0.png">

#### Video of Clicking on Chat Header

https://user-images.githubusercontent.com/16747822/111819334-7048e300-889d-11eb-94a3-306e3bef9a13.mov
